react/Logo/SeekLogo.js

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+
+const SeekLogo = ({ circleClass = 'sk-logo-circle', dotClass = 'sk-logo-dots' }) => (
+  <g>
+    <g className={circleClass}>
+      <circle cx="41.9" cy="41.9" strokeWidth="0" r="41.9" />
+    </g>
+    <g className={dotClass}>
+      <path d="m42.883 15.978c0-1.447 1.168-2.62 2.612-2.62 1.451 0 2.622 1.173 2.622 2.62 0 1.442-1.171 2.615-2.622 2.615-1.444 0-2.612-1.173-2.612-2.615z" />
```

Comment:
Creates a lot of jest snapshot noise, a possible alternative here would be dangerously setting on the `g`.